src/main/java/com/nucleus/config/AppConfig.java

```diff
@@ -69,19 +69,28 @@ public BasicDataSource getDataSource(){
         //ds.setUsername("nsbt");
         //ds.setPassword("qwerty");
         // Jigme's Oracle db credentials
-        /*
+        *//*
             ds.setUrl("jdbc:oracle:thin:@localhost:1521/pdborcl");
             ds.setUsername("nsbt");
             ds.setPassword("qwerty");
-        * */
+        * *//*
 
         ds.setUrl("jdbc:oracle:thin:@localhost:1521:ORCLCDB");
 
         ds.setUsername("c##username");
         ds.setPassword("pwd");
         return ds;
-    }
+    }*/
+   @Bean
```

Comment:
can you comment this function out, and comment back in the other one? 
Just add your connection details and comment those out in the other function